File: ios/Classes/tools/Tools.m
#import "Tools.h"
#define fileManager [NSFileManager defaultManager]

@implementation Tools

//Log
+ (void)log:(id)info{
    NSLog(@"CuriosityLog = %@", info);
}

+ (NSString *)resultInfo:(NSString *)info{
    return [NSString stringWithFormat:@"%@",info];
}

+ (NSString *)resultFail{
    return @"fail";
}
+ (NSString *)resultSuccess{
    return @"success";
}

+(BOOL) isImageFile:(NSString *)path{
    return [path hasSuffix:@".jpg"]
    ||[path hasSuffix:@".png"]
    ||[path hasSuffix:@".PNG"]
    ||[path hasSuffix:@".JPEG"]
    ||[path hasSuffix:@".JPG"]
    ||[path hasSuffix:@".GiF"]
    ||[path hasSuffix:@".gif"];
}
+(BOOL)isEmulator{
    if (TARGET_IPHONE_SIMULATOR == 1 && TARGET_OS_IPHONE == 1) {
        return YES;
    } else {
        return NO;
    }
}

@end

File: ios/Classes/Tools.swift
import Foundation

class Tools {
    static func log(_ info: Any?) {
        print("CuriosityLog =")
        print(info ?? "null")
    }

    static func resultInfo(_ info: String) -> String {
        info
    }

    static func resultFail(_ info: String) -> String {
        "fail"
    }

    static func resultSuccess(_ info: String) -> String {
        "success"
    }

    static func isEmulator() -> Bool {
        (TARGET_IPHONE_SIMULATOR == 1) && (TARGET_OS_IPHONE == 1)
    }

    static func isImageFile(_ path: String) -> Bool {
        path.hasSuffix(".jpg")
                || path.hasSuffix(".png")
                || path.hasSuffix(".PNG")
                || path.hasSuffix(".JPEG")
                || path.hasSuffix(".JPG")
                || path.hasSuffix(".GiF")
                || path.hasSuffix(".gif")
    }
}
